### PR TITLE
changesNotSentForReview support

### PR DESCRIPTION
**Pull request by princefr**
We added support for changesNotSentForReview

### CODE REVIEW

**Comment by tannermares** [comment]
Anything blocking this @r0adkll?

**Comment by Rukkaitto** [comment]
I need this too @r0adkll

**Comment by AMSTKO** [comment]
any news?

File: action.yml [hunk 1]
```diff
@@ -43,6 +43,9 @@ inputs:
   mappingFile:
     description: "The mapping.txt file used to de-obfuscate your stack traces from crash reports"
     required: false
+  changesNotSentForReview:
+    description: "If the build should be sent directly to release"
+    required: true
```
Comment (yairm210) [inline action.yml:48]: Adding a new "required" parameter will break all existing usages of this action.
This is probably one of the reasons this hasn't been accepted.
This should be changed to required:false, both here and below in main.ts, and the default value should enable the action to work as it has until now.

File: README.md [hunk 2]
```diff
@@ -109,11 +109,13 @@ with:
   userFraction: 0.33
   whatsNewDirectory: distribution/whatsnew
   mappingFile: app/build/outputs/mapping/release/mapping.txt
+  status: draft
+  changesNotSentForReview: true
```
Comment (yairm210) [inline README.md:113]: This doesn't make sense as an example usage. Draft shouldn't include userFraction as well. The draft should be a separate example.

File: README.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ The environment variable that is set when using the `track` `internalsharing`
 ## Example usage
 
 ```yaml
-uses: r0adkll/upload-google-play@v1
+uses: swootte/upload-google-play@V1.1.5
```
Comment (yairm210) [inline README.md:102]: The example usage of this repo won't have an example using another repo... and there's no reason to have it reference a specific subversion either

File: action.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: 'Upload Android Release to Play Store'
+name: 'Upload Android application to Play Store'
```
Comment (yairm210) [inline action.yml:1]: What's the reason behind this change?

File: src/main.ts [hunk 5]
```diff
@@ -114,6 +115,7 @@ async function run() {
             inAppUpdatePriority: inAppUpdatePriorityInt || 0,
             userFraction: userFractionFloat,
             status: status,
+            changesNotSentForReview: JSON.parse(changesNotSentForReview),
```
Comment (yairm210) [inline src/main.ts:118]: The JSON parsing should be in the validation step - if this is malformed (unparseable) json, we should warn the user

**Review by yairm210 — CHANGES_REQUESTED** [review]

**Comment by lozdan** [comment]
@princefr any update about this?

**Comment by boswelja** [comment]
@princefr are you still interested in working on this? If not, I'll take it over.

**Comment by boswelja** [comment]
Closes #84

**Comment by boswelja** [comment]
I have reimplemented this over on #117, closing this one.

**Pull request closed without merging**